France/nba_challenge.py

```diff
@@ -0,0 +1,96 @@
+has ridden James and Davis, along with a supporting cast built around them, \
+to the second-best record in the NBA. \nThe Lakers ended the season atop\
+the Western Conference with a record of 49-14. \nThey were narrowly behind \
+the Bucks for the best record in the league. \nDavis proved to the final \
+piece necessary for the Lakers to rebound from missing the playoffís last \
+year. \nLos Angeles was a dominant club on both sides of the ball and are \
+in a position to have another successful year next season. ')
+
+print('Challenge 3.2: Print out the paragraph but with only 1 sentence per line')
+# TODO: As above, orint out the paragraph with only 1 sentence per line, and all in upper case
+paragraph = 'The Lakers went all in this offseason and swung a deal for former \
+Pelicans forward Anthony Davis. \nThey sent a package of Brandon Ingram,\
+Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis. \
+\nThose three have made good developments with the Pelicans, especially \
+Brandon Ingram. \nBut the deal is still a huge win for the Lakers as Lebron\
+, Davis, and company have put together an incredible season. \nLos Angeles \
+has ridden James and Davis, along with a supporting cast built around them, \
+to the second-best record in the NBA. \nThe Lakers ended the season atop\
+the Western Conference with a record of 49-14. \nThey were narrowly behind \
+the Bucks for the best record in the league. \nDavis proved to the final \
+piece necessary for the Lakers to rebound from missing the playoffís last \
+year. \nLos Angeles was a dominant club on both sides of the ball and are \
+in a position to have another successful year next season. '
+print(paragraph.upper())
+print('Challenge 3.3: Make a boolean variable indicating whether you think the Lakers are the best NBA team')
+# TODO: make a variable called `lakers_are_best` to indicate this
+lakers_are_best = False
+# TODO: print out the variable in an f-string to convey your opinion on the lakers
+print(f'The statement that the Lakers are the best NBA team is \
+{lakers_are_best}.')
+print('Challenge 3.4: Type Conversion')
+# TODO: Convert your `lakers_are_best` variable to an integer, and print it out. 
+print(int(lakers_are_best))
+# TODO: Convert your `lakers_are best` variable to a float, and print it out
+print(float(lakers_are_best))
+print('Challenge 3.5: Type Conversion Part 2')
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to a string, then print it out.
+# convert to string
+print(str(jamal_murray_3pts_percentage))
+print(str(fred_vanvleet_3pts_percentage))
+print(str(james_harden_3pts_percentage))
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to an integer, then print it out.
+#convert to integer
+print(int(jamal_murray_3pts_percentage))
+print(int(fred_vanvleet_3pts_percentage))
+print(int(james_harden_3pts_percentage))
```

Comment:
I'm so proud of you for completing this challenge. This was not an easy challenge.